File: Meghana A S/Basics of Prgramming/DigitsOfANumber.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv){
    long int n;
    cin >> n;
    int arr[100];
    int i=0;
    while(n>0)
    {
        //cout<<n%10<<endl;
        arr[i]=n%10;
        i++;
        n=n/10;
    }
    for(int j=i;j>=0;j--)
    {
        cout<<arr[j]<<endl;
    }
    //write your code here
    
}

File: Tanisha/Basicsofprogramming/Gettingstarted/digits_of_a_number.java
package tanisha.basicsofprogramming.gettingstarted;
import java.util.*;
    
    public class digits_of_a_number{
    
    public static void main(String[] args) {
      // write your code here  
      
        Scanner scn = new Scanner(System.in);
        int n = scn.nextInt();
        char[] ch = String.valueOf(n).toCharArray();
        for (int i = 0; i < ch.length; i++) {
            System.out.println(ch[i]);
        }
        scn.close();
    }
}
